crates/ruff/resources/test/fixtures/pyflakes/F821_17.py

```diff
@@ -0,0 +1,102 @@
+"""Test type parameters and aliases"""
+
+# Type parameters in type alias statements
+
+from some_module import Bar
+
+type Foo[T] = T # OK
+type Foo[T] = list[T] # OK
+type Foo[T: Forward] = T # OK
+type Foo[*Ts] = Bar[Ts] # OK
+type Foo[**P] = Bar[P] # OK
+
+# Types used in aliased assignment must exist
+
+type Foo = DoesNotExist  # F821: Undefined name `DoesNotExist`
+type Foo = list[DoesNotExist]  # F821: Undefined name `DoesNotExist`
```

Comment:
Were you able to test these against the updated version of Pyflakes?

Reply:
I haven't! I'll do that and confirm they match. I tested for correctness with Pyright and Python runtime errors.

Reply:
The results are identical except that pyflakes emits two incorrect warnings

```
10:21: undefined name 'Ts'
11:21: undefined name 'P'
```

They appear to handle binding of type var tuples and param specs incorrectly.